### PR TITLE
Update arch-gnome.sh

### PR DESCRIPTION
**Pull request by yoge67855**

### CODE REVIEW

File: arch-gnome.sh [hunk 1]
```diff
@@ -150,7 +150,7 @@ install_xorg()
 		print_progress_text "Installing Xorg"
 		echo -e "If prompted to select provider(s), select default options"
 		echo ""
- 		sudo pacman -S xorg-server
+ 		sudo pacman -S xorg
```
Comment (drakkar1969) [inline arch-gnome.sh:153]: Based on the Arch Linux wiki (https://wiki.archlinux.org/index.php/Xorg), only the `xorg-server` package is required for a minimal install, not the whole `xorg` group. Other xorg packages can/should be installed as per need

File: arch-gnome.sh [hunk 2]
```diff
@@ -174,7 +174,7 @@ display_drivers()
 		print_progress_text "Installing nVidia video drivers"
 		echo -e "If prompted to select provider(s), select default options"
 		echo ""
-		sudo pacman -S nvidia lib32-virtualgl lib32-nvidia-utils
+		sudo pacman -S nvidia lib32-virtualgl nvidia-utils mesa nvidia-settings
```
Comment (drakkar1969) [inline arch-gnome.sh:177]: Thanks for the suggestions. Went back and looked at this:
* The `lib32-nvidia-utils` is effectively not required except for 32-bit (but will be pulled in by e.g. `steam` as a dependency).
* `lib32-virtualgl` is actually redundant
* `nvidia-utils` is a direct dependency of the `nvidia` package, and therefore doesn't need to be explicitly installed
* I hadn't included the `mesa` package here, as it is pulled in as a dependency during installation of the GNOME DE. However, it should be included here (for Intel support) if user decides not to install GNOME

So basically, the code would end up looking like this:
```shell
sudo pacman -S nvidia mesa
```

**Review by drakkar1969 — CHANGES_REQUESTED** [review]

File: arch-gnome.sh (hunk 1 above)
Comment (yoge67855) [reply arch-gnome.sh:153]: yeah i know but i always like installing full xorg group

File: arch-gnome.sh (hunk 2 above)
Comment (yoge67855) [reply arch-gnome.sh:177]: > The lib32-nvidia-utils is effectively not required except for 32-bit (but will be pulled in by e.g. steam as a dependency).


oh ok

> lib32-virtualgl is actually redundant


 yeah i forgot to remove it


> I hadn't included the mesa package here, as it is pulled in as a dependency during installation of the GNOME DE. However, it should be included here (for Intel support) if user decides not to install GNOME


i almost never install gnome i install xfce DE or WM
and mesa is needed as well for nvidia

**Comment by drakkar1969** [comment]
Closing with https://github.com/drakkar1969/arch-install/commit/d1db2002e1d6a2bea5e72c029968b50e1ee24887.

I see that you would like different options for installation, but ultimately these are scripts that need to be adapted to each user's needs/preferences. I would advise to fork the repo and modify as necessary.
